hazelcast/src/main/java/com/hazelcast/spi/merge/LatestAccessMergePolicy.java

```diff
@@ -26,24 +26,31 @@
  *
  * @since 3.10
  */
-public class LatestAccessMergePolicy extends AbstractMergePolicy {
+public class LatestAccessMergePolicy extends AbstractSplitBrainMergePolicy {
 
-    LatestAccessMergePolicy() {
+    public LatestAccessMergePolicy() {
     }
 
     @Override
-    public <K, V> V merge(SplitBrainMergeEntryView<K, V> mergingEntry, SplitBrainMergeEntryView<K, V> existingEntry) {
-        if (mergingEntry == null) {
-            return existingEntry.getValue();
+    public <V> V merge(MergeDataHolder<V> mergingData, MergeDataHolder<V> existingData) {
+        checkInstanceOf(mergingData, HitsDataHolder.class);
```

Comment:
Did you mean `LastAccessTimeDataHolder.class` here? Same in `LatestUpdateMergePolicy`

Reply:
Darn, this still works, since our `FullMergeDataHolderImpl` implements all these interfaces. Good catch, thanks!